wpilibNewCommands/src/main/native/include/frc2/command/CommandPtr.h

```diff
@@ -49,6 +49,16 @@ class CommandPtr final {
   [[nodiscard]]
   CommandPtr Repeatedly() &&;
 
+  /**
+   * Decorates this command to run repeatedly, restarting until the command runs
+   * for the given number of times. The decorated command can still be canceled.
+   *
+   * @param times the number of times to run the command
+   * @return the decorated command
+   */
+  [[nodiscard]]
+  CommandPtr Repeatedly(int times) &&;
```

Comment:
```suggestion
  CommandPtr Repeatedly(int repetitions) &&;
```
Don't forget to update the doc comment and the C++ source.